stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -45,13 +46,33 @@ public object ChatUI {
         }
 
     private var markdownOverride: ChatMarkdown? = null
-    private val defaultMarkdown: ChatMarkdown by lazy { ChatMarkdownImpl(appContext) }
+    private val defaultMarkdown: ChatMarkdown by lazy {
+        ChatMarkdown { textView, message ->
+            textView.text = message
+        }
+    }
+
+    @Deprecated(message = "ChatUI.markdown is deprecated. Use ChatUI.messageTextTransformer",
+        level = DeprecationLevel.WARNING)
```

Comment:
We should add `replaceWith` parameter into @Deprecated annotation.

Reply:
It won't be a direct method replacement so I don't add it. Otherwise, IDE will replace it incorrectly.

Reply:
I believe this should work correctly `replaceWith = ReplaceWith("ChatUI.messageTextTransformer", "io.getstream.chat.android.ui")`

Reply:
Done.